### PR TITLE
Recommend package installs & enhance installation & configuration docs

### LINKED ISSUE #196: Incorrect path to MySQL and PostgreSQL database schema files referenced in the installation instructions.
## Describe the bug

The [installation documentation](https://icinga.com/docs/icinga-reporting/latest/doc/02-Installation/) references `etc/schema/mysql.schema.sql` and `etc/schema/pgsql.schema.sql`.  The `etc` directory doesn't exist in the git repository.

## To Reproduce

1. Clone the project
2. cd into the cloned project
3. ls etc/schema/

## Expected behavior

I expected to see the schema files as referenced in documentation.

## Screenshots

N/A

## Your Environment

N/A

## Additional context

I believe the schema files are the ones found in the `schema` top  level directory in the project.  Either the documentation needs to be updated or the project top level directory `etc` needs to be created and the `schema` directory moved there.

### PR DESCRIPTION
**Pull request by yhabteab**
fixes #196

### CODE REVIEW

**Review by nilmerg — APPROVED** [review]
Seems fine

File: doc/03-Configuration.md [hunk 1]
```diff
@@ -25,9 +21,27 @@ that is used as the sender's address (From) in E-mails.
...
-reporting/templates  | Templates (create, edit, delete)
-reporting/timeframes | Timeframes (create, edit, delete)
+| Permission           | Applies to                        |
+|----------------------|-----------------------------------|
+| reporting/reports    | Reports (create, edit, delete)    |
+| reporting/schedules  | Schedules (create, edit, delete)  |
+| reporting/templates  | Templates (create, edit, delete)  |
+| reporting/timeframes | Timeframes (create, edit, delete) |
+
+## Icinga Reporting Daemon
```
Comment (lippserd) [inline doc/03-Configuration.md:31]: This section needs a little adjustment. Packages will install the systemd service automatically. And I would add `--now` to the `systemctl enable` command to start the daemon right away as it's periodically reloading the configuration anyway.

**Review by lippserd — CHANGES_REQUESTED** [review]

File: doc/03-Configuration.md [hunk 2]
```diff
@@ -25,9 +21,37 @@ that is used as the sender's address (From) in E-mails.
...
+
+> **Note**
+>
+> If you haven't installed this module from packages, you have to configure this as a `systemd` service yourself by just
+> copying the example service definition from `/usr/share/icingaweb2/modules/reporting/config/systemd/icinga-reporting.service`
+> to `/etc/systemd/system/icinga-reporting.service`.
+
+<!-- {% endif %} -->
+
+Afterwards, you can run the following command to enable and start the daemon.
```
Comment (nilmerg) [inline doc/03-Configuration.md:53]: > Afterwards,

 After what?

**Review by nilmerg — CHANGES_REQUESTED** [review]

**Review by nilmerg — DISMISSED** [review]

**Review by nilmerg — APPROVED** [review]

File: doc/03-Configuration.md [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
 
-1. [Backend](#backend)
-2. [Mail](#mail)
-3. [Permissions](#permissions)
+Icinga Reporting is configured via the web interface. Below you will find an overview of the necessary settings.
 
 ## Backend
 
-If not already done during the installation of Icinga Reporting, setup the reporting database backend now.
+Configuring the Icinga Reporting database backend now.
```
Comment (nilmerg) [inline doc/03-Configuration.md:7]: ~~Configuring the Icinga Reporting~~
That should not be required to repeat here.

Instead, tell what's stored inside the database if you want a description for this section.

File: doc/03-Configuration.md [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
 
-Create a new [Icinga Web 2 resource](https://icinga.com/docs/icingaweb2/latest/doc/04-Resources/#database)
-for [Icinga Reporting's database](https://icinga.com/docs/icinga-reporting/latest/doc/02-Installation/#database-setup)
-using the `Configuration -> Application -> Resources` menu.
+1. Create a new resource for the Icinga Reporting via the `Configuration -> Application -> Resources` menu.
```
Comment (nilmerg) [inline doc/03-Configuration.md:9]: ~~the~~ Icinga Reporting

File: doc/03-Configuration.md [hunk 5]
```diff
@@ -1,20 +1,16 @@
...
 
-Then tell Icinga Reporting which database resource to use. This can be done in
-`Configuration -> Modules -> reporting -> Backend`. If you've used `reporting`
-as name for the resource, this is optional.
+2. Configure the resource you just created as the database connection for the Icinga Reporting module using the
```
Comment (nilmerg) [inline doc/03-Configuration.md:11]: ~~the~~ Icinga Reporting ~~module~~

**Review by nilmerg — CHANGES_REQUESTED** [review]

**Review by nilmerg — APPROVED** [review]
I've also tested this locally as if rendered on the website. Looks fine, albeit the notice in 03-Configuration.md about the install without packages will not be hidden there, as the template macros are not evaluated in this context. But that's not a problem here.

**Pull request merged**